### PR TITLE
add parameters to table and image viewers

### PR DESCRIPTION
**Pull request by stargaser**
```
- add center parameters for 2MASS survey images
- enable image and table urls or paths
- connect to specified Firefly URL
- add colorbar functionality
- clarify instructions in README
```

### CODE REVIEW

File: js/src/Image.js [hunk 1]
```diff
@@ -34,20 +34,33 @@ var ImageView = widgets.DOMWidgetView.extend({
...
             AllowImageSelection : true
         };
-        this.el.id = `imageViewer-${seq++}`;
-        this.model.on('change:GridOn change:SurveyKey', this.redraw, this);
+        this.model.on('change:GridOn change:SurveyKey change:FilePath', this.redraw, this);
+        this.model.on('change:colorbar', this.update_color, this);
         this.redraw = this.redraw.bind(this);
+        this.update_color = this.update_color.bind(this);
         setTimeout(this.redraw, 0);
+        setTimeout(this.update_color, 0);
```
Comment (loitly) [inline js/src/Image.js:42]: Keep in mind that the original sample code has everything in the 'render' function.  It also implemented redraw inefficiently.  Eventually, we should go back move the logic into its proper places and implement redraw similar to update_color.

**Review by loitly — APPROVED** [review]
mytable = tv.selection() fail with ValueError: unknown url type: 
Could be changes in Firefly that causes this.  I am not sure.
The changes look fine to me.

File: js/src/Image.js (hunk 1 above)
Comment (stargaser) [reply js/src/Image.js:42]: Good point. I'll make a new ticket for this refactoring.

**Comment by stargaser** [comment]
mytable = tv.selection() will fail if the cells are executed too quickly, in my experience. Maybe it takes time for the server to apply a selection to the table.

**Pull request merged**